src/cli/commands/run.js

```diff
@@ -68,7 +68,7 @@ export async function run(
     for (const action of actions) {
       const cmd = scripts[action];
       if (cmd) {
-        const isWin = /win/.test(process.platform);
+        const isWin = /win32/.test(process.platform);
```

Comment:
Could this just be `process.platform === 'win32'`? Looks like that's what Node.js itself does, so it should be safe: https://github.com/nodejs/node/blob/db1087c9757c31a82c50a1eba368d8cba95b57d0/lib/os.js#L6